content/docs/standard-library/core/_index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Core
+weight: 5
+---
+The `Core`  section describes the core component's.
+
+# Emitter
+The `Emitter` constantly emits a message that you bind to it using the compiler directive `#bind`.This should be avoided if possible (use const's and literals instead).
+```
+pub component Emitter<T>() (msg T)
+```
+
+### Usage
+
+# Destructor
```

Comment:
Please add explanation what this is.

Destructor is a component that needs to destroy unwanted message. Compiler will implicitly insert destructors for every unused outport in component's network so there's usually no need to use it explicitly.